packages/applications/ssr/src/app/laureats/[identifiant]/achevement/attestation-conformite:transmettre/page.tsx

```diff
@@ -42,15 +42,17 @@ export default async function Page({ params: { identifiant } }: IdentifiantParam
         },
       });
 
+    const peutVoirMainlevée = Option.isSome(garantiesFinancières);
+
     const peutDemanderMainlevée =
-      Option.isSome(garantiesFinancières) &&
-      garantiesFinancières.garantiesFinancières.attestation !== undefined;
+      peutVoirMainlevée && garantiesFinancières.garantiesFinancières.attestation !== undefined;
```

Comment:
Pourquoi pas juste : 

```suggestion
      peutVoirMainlevée && garantiesFinancières.garantiesFinancières.attestation
```

Reply:
Alors ce n'est pas un boolean donc ça serait plutôt `peutVoirMainlevée && !!garantiesFinancières.garantiesFinancières.attestation`, j'imagine que le `garantiesFinancières.garantiesFinancières.attestation !== undefined` est plus lisible non ?

Reply:
Oui l'un ou l'autre c'est ok, mais on peut laisser tel quel c'est bien